core/src/rpc.rs

```diff
@@ -1746,13 +1755,22 @@ impl JsonRpcRequestProcessor {
             .account_indexes
```

Comment:
I think the way you did this is ok. However, the .account_indexes is specified in the config to rpc also from validator. I specifically had to chop out the include/exclude info. So, you could just keep a copy of the include/exclude info in rpc directly. There are 2 sources of truth then, but we already do that with account_indexes.

Reply:
[here](https://github.com/solana-labs/solana/blob/597373f5fa8037cee1a1241e6dc3bf1f9e1631d7/validator/src/main.rs#L2119) is where I exclude the include/exclude info

Reply:
Ah, I failed to track where the verbose/concise account_indexes were split. Yeah, let's do that; saves any runtime helpers needed.

Reply:
@jeffwashington done here: https://github.com/solana-labs/solana/pull/17193/commits/32118afe7511c600371a45f9b947727ec0dbc4b8
Can you take a look?

I considered changing the type to `Arc<AccountSecondaryIndexes>` to reduce to only one source of truth, but the level of churn was very high. Let me know if you prefer that, though.

Reply:
perfect.
I prefer not an arc myself. The secondary config is created once at startup and never changed and looked up on each modification to the accounts index. 2 copies is ok with me.